docs/user/dashboard/aggregation-based.asciidoc

```diff
@@ -0,0 +1,216 @@
+
+| image:images/area.png[Area chart]
+
+a| *Data table*
+
+Displays your aggregation results in a tabular format. Use data tables to display server configuration details, track counts, min,
+or max values for a specific field, and monitor the status of key services.
+
+| image:images/data_table.png[Data table]
+
+a| *Gauge*
+
+Displays your data along a scale that changes color according to where your data falls on the expected scale. Use the gauge to show how metric
+values relate to reference threshold values, or determine how a specified field is performing versus how it is expected to perform.
+
+| image:images/gauge.png[Gauge]
+
+a| *Goal*
+
+Displays how your metric progresses toward a fixed goal. Use the goal to display an easy to read visual of the status of your goal progression.
+
+| image:images/goal.png[Goal]
+
+a| *Heat map*
+
+Displays graphical representations of data where the individual values are represented by colors. Use heat maps when your data set includes
+categorical data. For example, use a heat map to see the flights of origin countries compared to destination countries using the sample flight data.
+
+| image:images/heat_map.png[Heat map]
+
+a| *Horizontal Bar*
+
+Displays bars side-by-side where each bar represents a category. Use bar charts to compare data across a
+large number of categories, display data that includes categories with negative values, and easily identify
+the categories that represent the highest and lowest values. {kib} also supports vertical bar charts.
+
+| image:images/bar.png[Bar chart]
+
+a| *Line*
+
+Displays data points that are connected by a line. Use line charts to visualize a sequence of values, discover
+trends over time, and forecast future values.
+
+| image:images/line.png[Line chart]
+
+a| *Metric*
+
+Displays a single numeric value for an aggregation. Use the metric visualization when you have a numeric value that is powerful enough to tell
+a story about your data.
+
+| image:images/metric.png[Metric]
+
+a| *Pie*
+
+Displays slices that represent a data category, where the slice size is proportional to the quantity it represents.
+Use pie charts to show comparisons between multiple categories, illustrate the dominance of one category over others,
+and show percentage or proportional data.
+
+| image:images/pie.png[Pie chart]
+
+a| *Tag cloud*
+
+Graphical representations of how frequently a word appears in the source text. Use tag clouds to easily produce a summary of large documents and
+create visual art for a specific topic.
+
+| image:images/tag_cloud.png[Tag cloud]
+
+|===
+
+[float]
+[[create-aggregation-based-panel]]
+=== Create an aggregation-based panel
+
+Choose the type of panel you want to create, then use the editor to configure the options. Each panel type supports different options. 
+
+. From the dashboard, click *Create new*, then click *Aggregation based* on the *New visualization* window.
```

Comment:
We've renamed a lot of the "Create new" buttons.  I couldn't find this on the dashboard.